asf_mission_data_tool/pipeline/uk_territorial_greenhouse_gas_emissions_statistics_pipeline/get_uk_territorial_greenhouse_gas_emissions_statistics_to_bronze.py

```diff
@@ -0,0 +1,24 @@
+from asf_mission_data_tool.getters.data_getters import (
+    save_to_s3_bronze,
+    get_latest_version,
+    append_file_bronze_to_latest_version,
+)
+
+dataset_name = "uk_territorial_greenhouse_gas_emissions_statistics"
+
+latest_version = get_latest_version(dataset_name)
```

Comment:
```suggestion
latest_version = get_latest_version(dataset_name, filter="final")
```

Also wanted to note that the most recent dataset is provisional for 2023. If we want the most recent "final" version (which have different files and tables) then we can pass this filter argument.